Bejebeje.DataAccess/Migrations/BbContextModelSnapshot.cs

```diff
@@ -3,189 +3,224 @@
 using Bejebeje.DataAccess.Context;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
 
 namespace Bejebeje.DataAccess.Migrations
 {
-  [DbContext(typeof(BbContext))]
-  partial class BbContextModelSnapshot : ModelSnapshot
-  {
-    protected override void BuildModel(ModelBuilder modelBuilder)
+    [DbContext(typeof(BbContext))]
+    partial class BbContextModelSnapshot : ModelSnapshot
     {
```

Comment:
Same as in Bejebeje.DataAccess/Migrations/20190511151840_InitialCreate.cs I'm confused.